src/Components/NavBar/NavBar.tsx

```diff
@@ -64,11 +64,13 @@ const NavBar: React.FC<NavBarProps> = () => {
           </div>
         ) : (
           <div className={styles.buttonList}>
-            <button
-              className={`${styles.navBarButton} ${styles.profileButton}`}
-            >
-              {userInfo.username}
-            </button>
+            <Link to={"/mypage?tab=profile"}>
+              <button
+                className={`${styles.navBarButton} ${styles.profileButton}`}
+              >
+                {userInfo.username}
+              </button>
+            </Link>
```

Comment:
굳이 Link, button 중복해서 쓸 필요 없을 듯합니다.
```suggestion
            <Link to={"/mypage?tab=profile"}
                className={`${styles.navBarButton} ${styles.profileButton}`}
              >
                {userInfo.username}
            </Link>
```